gradle/publish.gradle

```diff
@@ -33,6 +41,20 @@ githubPages {
     }
 }
 
+pluginBundle {
+    website = WEBSITE
+    vcsUrl = WEBSITE
+    description = 'A Gradle plugin to easily apply the same setup of static analysis tools across different Android, Java or Kotlin projects.'
+    tags = ['java', 'kotlin', 'android', 'static analysis', 'code quality']
+
+    plugins {
+        gradleStaticAnalysisPlugin {
+            id = "$GROUP_ID.$ARTIFACT_ID"
```

Comment:
Same comment here. 
Also in jCenter, we have the full name `gradle-static-analysis-plugin` as id because jCenter contains all libraries. So having gradle and plugin names make sense. 

But Gradle Plugin Portal only consist of those. So we should really have just `com.novoda.static-analysis`

Reply:
that makes sense 👍